### DISCUSSION TITLE
NX support

### DISCUSSION THREAD

**Discussion by ffMathy** [oxc-project/oxc#3271] [Feature Request]
It'd be great if I could use Oxlint instead of ESLint for my NX projects.

**Comment by andymac4182** [comment]
Curious what doesn't work with nx?

**Comment by ffMathy** [comment]
It's just that there isn't any oxlint executor for nx.

**Comment by andymac4182** [comment]
Is there any docs on what that looks like? Sorry not super familiar with nx.

**Comment by ffMathy** [comment]
Here are some docs for it:
https://nx.dev/extending-nx/recipes/local-executors

Essentially it'll allow an NX project to be executed by Oxlint. You can probably get inspired by how the Eslint executor is here: https://github.com/nrwl/nx/tree/master/packages/eslint/src/executors/lint

**Comment by the-ult** [comment]
This would indeed be great. To be able to switch to oxc by default.

Currently using it with Nx would mean that with each new library, you would manually need to swap it with oxc.
And you can't use the default `nx lint` command.

**Comment by Nas3nmann** [comment]
I just release a first version of an oxlint nx community plugin. You can try it here: https://github.com/Nas3nmann/nx-oxlint
